src/tests/issues.targets

```diff
@@ -1529,6 +1529,10 @@
             <Issue>Static virtual methods are not yet implemented in the Mono runtime.</Issue>
         </ExcludeList>
 
+        <ExcludeList Include="$(XunitTestBinBase)/Loader/classloader/InlineArray/**">
+            <Issue>InlineArray support is not yet implemented in the Mono runtime.</Issue>
```

Comment:
Please file an issue for this.

/cc @lambdageek

Reply:
There is an issue already -[ [Tracking] Mono support for InlineArrayAttribute](https://github.com/dotnet/runtime/issues/80798)

I will add a link to the exclusion.